app/index.js

```diff
@@ -410,7 +410,7 @@ function installDevExtensions(isDev) {
   if (!isDev) {
     return Promise.resolve();
   }
-  const installer = require('electron-devtools-installer'); // eslint-disable-line global-require
+  const installer = require('electron-devtools-installer'); // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
It could be a slightly better idea to, instead of turning it off, changing the config of the rule at that position, to turn on the `devDependencies` option. That way, if someone removes it from the project while it's still being used, you'd still get the error here.

It's a bit akin to nitpicking as it's a very edge case probably, but I think it's a worthwhile change.